### PR TITLE
Updated http cacher to use .ToUniversalTime()

### PR DESCRIPTION
**Pull request by psantiago**
I started getting a bunch of broken cache tests locally and on our build servers. Probably some new windows and/or .net update bug?  The build server just had updates applied, and the tests just started breaking there - I've been getting these failing tests for a few weeks, but just ignored them since they built fine on the server (and just assumed I has something misconfigured locally).

Appending .ToUniversalTime() seems to fix all the things, and all the tests in the project still seem to pass (so I'm assuming I didn't break anything else). 

Also, it's recommended to use universal times anyway - https://msdn.microsoft.com/en-us/library/4y13wyk9(v=vs.110).aspx.

In conclusion: 
fixitfixitfixit

### CODE REVIEW

**Comment by jdehlin** [comment]
Looks good to me. Thanks for the PR!

**Comment by psantiago** [comment]
@jdehlin - can you push out some nuget packages too (including the strongnamed one)? Feel free to shove -beta or -unstable or whatever if you want. Thanks!

**Comment by jdehlin** [comment]
NuGet packages published with you change.

**Comment by psantiago** [comment]
@jdehlin if I look at the decompiled version of the new nuget packages the new change doesn't seem to be in them:

oh_noes

**Comment by jdehlin** [comment]
Sorry about that. New version uploaded that actually has the changes this time :)
